### PR TITLE
Question 1 complete

### PR DESCRIPTION
**Pull request by Akin-23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
-# Northcoders News API
+# Akin News API
+
+In order to successfully connect to the two databases locally , you have to create two .env files in this project : .env.test and .env.development. Into each, add PGDATABASE='databasename', with the correct database name for that environment (see /db/setup.sql for the database names).
+
```
Comment (2202Hannah) [inline README.md:4]: You can be more specific include the database names here if you like.

File: __tests__/api-test.test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topics to the client", () => {
+    return request(app).get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toEqual(expect.any(Array));
+        expect(topics.length).toBe(3);
```
Comment (2202Hannah) [inline __tests__/api-test.test.js:20]: Really nice testing and it's good to see that you remembered to check for length of the array too.

**Review by 2202Hannah — APPROVED** [review]
Remember to separate your work on to separate branches for each task going forward. This pull request is for both task 1 & 2 but as they are small tasks I have reviewed them together on this occasion.

**Review by B7Bprog — APPROVED** [review]
Please merge this.

**Pull request merged**